<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kaname</class>
 <widget class="QMainWindow" name="Kaname">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kaname</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Marked Objects</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QListWidget" name="listMarkedObjects">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="MarkingWidget" name="marker">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Image Info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Image Size:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Next Object</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leImageName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leImageSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>File:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="leNextObj">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Key Bindings:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QTextEdit" name="textEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Edit Object Names: Ctrl + E&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Add Images: Ctrl + A&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Clear Images: Ctrl + W&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Load Images: Ctrl + Q&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Save Labeling Data: Ctrl + S&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Previous Image: A&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Next Image: D&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Skip Current Object: X&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Undo Current Mark: Z&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;About: ?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_EditObjectNames"/>
   <addaction name="separator"/>
   <addaction name="action_AddImages"/>
   <addaction name="action_LoadImages"/>
   <addaction name="action_ClearImages"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_PreviousImage"/>
   <addaction name="action_NextImage"/>
   <addaction name="separator"/>
   <addaction name="action_NextObject"/>
   <addaction name="action_UndoCurrentMark"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
  </widget>
  <action name="action_AddImages">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/add.png</normaloff>:/icon/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Image(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_ClearImages">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/clear.png</normaloff>:/icon/clear.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear Images</string>
   </property>
   <property name="toolTip">
    <string>Clear All Images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_NextImage">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/right.png</normaloff>:/icon/right.png</iconset>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
   <property name="toolTip">
    <string>Next Image</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="action_PreviousImage">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/left.png</normaloff>:/icon/left.png</iconset>
   </property>
   <property name="text">
    <string>Previous Image</string>
   </property>
   <property name="toolTip">
    <string>Previous Image</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="action_NextObject">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/next.png</normaloff>:/icon/next.png</iconset>
   </property>
   <property name="text">
    <string>Next Object</string>
   </property>
   <property name="toolTip">
    <string>Next Object to Mark</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="action_UndoCurrentMark">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/undo.png</normaloff>:/icon/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo Current Mark</string>
   </property>
   <property name="toolTip">
    <string>Undo Current Mark</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="action_LoadImages">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/check.png</normaloff>:/icon/check.png</iconset>
   </property>
   <property name="text">
    <string>Load Images</string>
   </property>
   <property name="toolTip">
    <string>Load Images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/save.png</normaloff>:/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Labeling Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_EditObjectNames">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/list.png</normaloff>:/icon/list.png</iconset>
   </property>
   <property name="text">
    <string>Edit Object Names</string>
   </property>
   <property name="toolTip">
    <string>Edit object names</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/appbar.question.png</normaloff>:/icon/appbar.question.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>?</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MarkingWidget</class>
   <extends>QFrame</extends>
   <header>MarkingWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
